lib/packages/util/CHANGELOG.md

```diff
@@ -14,7 +14,10 @@ All notable changes to this project will be documented in this file.
 - Added Base64 conversion (URI and UTF-8 compatible)
     - `convertStringToBase64String`
     - `convertBase64StringToString`
-    
+- New Symbols for BURST (Ƀ) and Planck (ƀ)
```

Comment:
Planck symbol is the minuscule version of the Ƀ...Open for discussion.... there is another symbol ꝑ which is a good candidate, too